etc/notebooks/sentiment_boilerplate_demo/sentiment_boilerplate_demo.ipynb

```diff
@@ -0,0 +1,797 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "urth": {
+     "dashboard": {
+      "hidden": true
+     }
+    }
+   },
+   "source": [
+    "#Sentiment Boilerplate\n",
+    "\n",
+    "This notebook provides a boilerplate for displaying and aggregating metrics about sentiment into a dashboard. In this notebook sentiment is defined as a positive or negative classification for some metric.\n",
+    "\n",
+    "**Table of Contents**\n",
+    "\n",
+    "1. [Getting Started](#Getting-Started)\n",
+    "1. [Extension Point](#Extension-Point)\n",
+    "1. [Boilerplate Code](#Boilerplate)\n",
+    "    1. [Utilities](#Utilities)\n",
+    "    1. [get_dashboard_data](#get_dashboard_data)\n",
+    "    1. [get_historical_sentiment_data](#get_historical_sentiment_data)\n",
+    "1. [UI](#UI)\n",
+    "    1. [Styles](#Styles)\n",
+    "    1. [Connecting To The Backend](#Connecting-To-The-Backend)\n",
+    "    1. [Dashboard Header](#Dashboard-Header)\n",
+    "    1. [Overall Sentiment Widget](#Overall-Sentiment-Widget)\n",
+    "    1. [Sentiment Breakdown Widget](#Sentiment-Breakdown-Widget)\n",
+    "    1. [Historical Sentiment Widget](#Historical-Sentiment-Widget)\n",
+    "\n",
+    "\n",
+    "\n",
+    "##Getting Started\n",
+    "\n",
+    "This boilerplate contains a simple implementation of [`get_sentiment_counts_during`](#Extension-Point) to illustrate the data structure needed for the boilerplate UI. This is the extension point where one would adopt this dashboard to their particular use case. Returning the proper data structure from the `get_sentiment_counts_during` function is the only requirement needed to integrate with this dashboard."
```

Comment:
Before telling people to change the function, maybe get them to run the notebook as is to see what it looks like.

Also, I got thrown off a bit by the word "integrate" here, but not sure what I would say instead. "You have to implement this one function to make this dashboard show your own sentiment data?"

Feels like that's the key point to stress again and again: you don't have to reimplement the UI, just the data binding.